streams.yml

```diff
@@ -65,3 +65,13 @@ rhel-7-golang:
   # tests that don't happen downstream.
   transform: rhel-7/golang
   upstream_image: registry.ci.openshift.org/ocp/builder:rhel-7-golang-1.16-openshift-{MAJOR}.{MINOR}
+
+rhel8:
+  image: registry.redhat.io/ubi8:latest
+  upstream_image: registry.ci.openshift.org/ocp/builder:rhel-8-openshift-{MAJOR}.{MINOR}.art
```

Comment:
Should match `registry.ci.openshift.org/ocp/builder:rhel-8-base-openshift-{MAJOR}.{MINOR}` ?